### PR TITLE
update flipper admins

### PR DESCRIPTION
**Pull request by erikphansen**
## Original issue(s)
department-of-veterans-affairs/va.gov-team#0000

### CODE REVIEW

File: config/settings.yml [hunk 1]
```diff
@@ -665,7 +665,6 @@ flipper:
     - adewitt@thoughtworks.com
     - alastair@adhocteam.us
     - anna@adhoc.team
-    - bconley@governmentcio.com
```
Comment (erikphansen) [inline config/settings.yml:668]: left gcio a while ago

File: config/settings.yml [hunk 2]
```diff
@@ -689,7 +688,6 @@ flipper:
     - kabrown@thoughtworks.com
     - kadams@governmentcio.com
     - kam@adhocteam.us
-    - keifer@oddball.io
```
Comment (erikphansen) [inline config/settings.yml:692]: left oddball a while ago

File: config/settings.yml [hunk 3]
```diff
@@ -713,15 +711,13 @@ flipper:
     - patrick@adhocteam.us
     - paul.phillips@thoughtworks.com
     - pherbert@thoughtworks.com
-    - philip.becker@oddball.io
```
Comment (erikphansen) [inline config/settings.yml:716]: left the project as far as I know

File: config/settings.yml [hunk 4]
```diff
@@ -713,15 +711,13 @@ flipper:
...
     - regarr@gmail.com # Robin Garrison - AdHoc
     - rian.fowler@adhoc.team
     - rianfowler@outlook.com
     - roth_matthew@bah.com
     - ryan.thurlwell@va.gov
     - saman.moshafi@thoughtworks.com
     - samara.strauss@va.gov
-    - shallie@governmentcio.com
```
Comment (erikphansen) [inline config/settings.yml:724]: will leave gcio EOW

**Review by LindseySaari — APPROVED** [review]

**Review by LindseySaari — APPROVED** [review]

**Pull request merged**